src/components/ohri-form-launcher/ohri-form-launcher.component.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Add } from "@carbon/react/icons";
+import styles from "./launcher-with-intent.scss";
+import { useTranslation } from "react-i18next";
+import { OHRIOverflowMenu } from "../overflow-menu-button/ohri-overflow-menu.component";
+
+export const OHRIFormLauncherWithIntent: React.FC<{
```

Comment:
Nothing such be named **ohri**
Rename to **ugandaemr-form-...** or keep it at **form-launcher....**

Reply:
This was adjusted.